PhysicsTools/PatAlgos/python/cleaningLayer1/tauCleaner_cfi.py

```diff
@@ -8,7 +8,7 @@
         'tauID("decayModeFinding") > 0.5 &'
         ' tauID("byLooseCombinedIsolationDeltaBetaCorr3Hits") > 0.5 &'
         ' tauID("againstMuonTight3") > 0.5 &'
-        ' tauID("againstElectronVLooseMVA5") > 0.5'
+        ' tauID("againstElectronVLooseMVA6") > 0.5'
```

Comment:
Hi Roger,

It would be nice to have a single source of settings here and in the PhysicsTools/PatAlgos/python/tools/tauTools.py

It looks like PATTauCleaner is not running in miniAOD setup. So, this is an item not affecting the production workflows.